src/Akeneo/CouplingDetector/Configuration/Configuration.php

```diff
@@ -16,23 +17,24 @@ class Configuration
     /** @var RuleInterface[] */
     private $rules;
 
+    /** @var ExclusionInterface[] */
+    private $exclusions;
+
     /** @var Finder */
     private $finder;
 
     /**
      * Configuration constructor.
      *
-     * @param RuleInterface[] $rules
-     * @param Finder          $finder
+     * @param Finder               $finder
+     * @param RuleInterface[]      $rules
+     * @param ExclusionInterface[] $exclusions
      */
-    public function __construct(array $rules, Finder $finder = null)
+    public function __construct(Finder $finder, array $rules, array $exclusions = [])
```

Comment:
yes sure, did a try with extra conf to avoid to "pollute" rules but still not sure of the approach